third_party/bigframes_vendored/pandas/core/common.py

```diff
@@ -0,0 +1,44 @@
+# Contains code from https://github.com/pandas-dev/pandas/blob/main/pandas/core/common.py
+from __future__ import annotations
+
+from typing import Callable, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from third_party.bigframes_vendored.pandas.pandas._typing import T
+
+
+def pipe(
+    obj, func: Callable[..., T] | tuple[Callable[..., T], str], *args, **kwargs
+) -> T:
+    """
+    Apply a function ``func`` to object ``obj`` either by passing obj as the
+    first argument to the function or, in the case that the func is a tuple,
+    interpret the first element of the tuple as a function and pass the obj to
+    that function as a keyword argument whose key is the value of the second
+    element of the tuple.
+
+    Parameters
```

Comment:
google style docstring please.

Also could you please screenshots of the API doc rendering?

Reply:
Fixed to follow google style docstrings.

Reply:
Examples are not rendering (they go before Args/Returns). also the args and kwargs appear funky https://screenshot.googleplex.com/97cktrgpWUyMzxL

Reply:
fixed docstring. haven't got the rendering process figure out yet though

Reply:
Thanks! [nit] The style in this file (common.py) itself is not google style, but it isn't leading to the website as the one in generic.py is taking effect, so we can remove it from here, or update to google style for consistency.

Reply:
sorry, forgot that I had two docstrings to fix. changed this one to google style too